### DISCUSSION TITLE
Error 126: Could not open Dll "python***.dll"

### DISCUSSION THREAD

**Discussion by coolmyf** [pyscripter/python4delphi#476] [Q&A] [answered]
When I compile the application in delphi or lazarus, the same error message as the title always be  shown. I have set pythonengine's dllname and dllpath property, but the problem still exists. Could  anyone tell me to do? Thanks!

**Comment by pyscripter** [comment]
Are you on Windows?
Have you got python installed?  Which version?  
Is it the same bitness as your Lazarus app? (32 bits or 64 bits)?

**Reply by coolmyf** [reply]
Yes. I am using windows 7. Lazarus version is 2.2.0 and FPC version is 3.2.2. The lazarus is a 32bit verison. I have python 3.8.10 (64bit) installed on my PC.Besides, I also have a similar problem when I use delphi xe10 seattle on the same PC. Do you mean that I must use Lazarus and Python the same 32 or 64 bit version?

**Comment by pyscripter** [comment, marked as answer]
Yes.

**Reply by coolmyf** [reply]
Thanks!
